docs/variables.md

```diff
@@ -1484,8 +1562,14 @@ but also flushes all L0 files.
 | Data type    | Boolean                            |
 | Default      | OFF                                |
 
+!!! note
+
+    In version [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md) and later, the default value is changed from `ON` to `OFF`.
+
 Forces MyRocks to immediately flush all memtables out to data files.
 
+The default value is `OFF` which means the variable is disabled by default.
```

Comment:
This variable is a switch. We call `rocksdb_force_flush_memtable_now=ON` to immediately flush all memtables and later it is automatically set to OFF.

Reply:
updated